### PR TITLE
open batch changes as external link

### PR DESCRIPTION
**Pull request by 5h1rU**
Batch Changes now opens in a new tab

### CODE REVIEW

**Review by eseliger — APPROVED** [review]
LGTM thank you :)

File: client/web/src/nav/GlobalNavbar.tsx [hunk 1]
```diff
@@ -246,7 +246,9 @@ export const GlobalNavbar: React.FunctionComponent<Props> = ({
                             <NavLink to="/code-monitoring">Monitoring</NavLink>
                         </NavItem>
                         <NavItem icon={BatchChangesIconNav}>
-                            <NavLink to="/batch-changes">Batch Changes</NavLink>
+                            <NavLink to="https://about.sourcegraph.com/batch-changes/" external={true}>
+                                Batch Changes
+                            </NavLink>
```
Comment (limitedmage) [inline client/web/src/nav/GlobalNavbar.tsx:251]: This should only happen on .com, right? on self-hosted instances we still need to be able to point to the internal batch changes page.

Comment (eseliger) [reply client/web/src/nav/GlobalNavbar.tsx:251]: Oh no you're right! I'll fix it

Comment (eseliger) [reply client/web/src/nav/GlobalNavbar.tsx:251]: https://github.com/sourcegraph/sourcegraph/pull/21729

**Pull request merged**